mobile/android/app/build.gradle

```diff
@@ -6,10 +11,10 @@ if (localPropertiesFile.exists()) {
     }
 }
 
-def flutterRoot = localProperties.getProperty('flutter.sdk')
```

Comment:
why commented these instead of removing when they will not be used?

Reply:
@ua741